model/Core/Classes/Agent.md

```diff
@@ -19,3 +19,15 @@ This is not to be confused with tools that are used to perform tasks.
 - name: Agent
 - SubclassOf: Element
 - Instantiability: Concrete
+
+## Summary @ja
```

Comment:
I noticed this has `@ja` - the next file has `@jp`

Reply:
Fixed by the most recent commit